src/main/fc/cli.c

```diff
@@ -1759,6 +1759,81 @@ static void cliSerial(char *cmdline)
     memcpy(currentConfig, &portConfig, sizeof(portConfig));
 }
 
+#ifndef SKIP_SERIAL_PASSTHROUGH
```

Comment:
For consistency with other features this should be renamed to `USE_SERIAL_PASSTHROUGH`.